frontend/src/app/services/api.service.ts

```diff
@@ -256,6 +257,22 @@ export class ApiService {
         return this.http.get<CorpusDocumentationPage>(url);
     }
 
+    public createCorpusDocumentationPage(data: CorpusDocumentationPageSubmitData) {
+        const url = this.apiRoute(this.corpusApiUrl, 'documentation/');
+        return this.http.post(url, data);
+    }
+
+    public updateCorpusDocumentationPage(pageID: number, data: CorpusDocumentationPageSubmitData) {
+        const url = this.apiRoute(this.corpusApiUrl, `documentation/${pageID}/`);
+        return this.http.put(url, data);
+    }
+
+    public deleteCorpusDocumentationPage(pageID: number) {
+        const url = this.apiRoute(this.corpusApiUrl, `documentation/${pageID}/`);
+        return this.http.delete(url);
+    }
```

Comment:
We should really build some generic REST endpoints for the API service at some point..

Reply:
Good point, I'll make an issue.